app/controllers/catalog_controller.rb

```diff
@@ -115,9 +116,7 @@ class CatalogController < ApplicationController
     config.add_facet_field 'campus_facet', label: 'Campus', sort: 'index', limit: -1, single: true
     config.add_facet_field 'library_facet', label: 'Library', sort: 'index', show: false, limit: -1, single: true # just advanced search
     config.add_facet_field 'up_library_facet', label: 'University Park Libraries', sort: 'index', limit: -1, single: true
-    config.add_facet_field 'pub_date_itsi', label: 'Publication Year', range: {
-      segments: false
-    }
+    config.add_facet_field 'pub_date_sort_itsi', label: 'Publication Year', range: { segments: false }
```

Comment:
@cdmo we renamed the field as `pub_date_itsi` ;)

Reply:
man facepalming.